java/vector/src/main/java/org/apache/arrow/vector/ViewVarCharVector.java

```diff
@@ -279,6 +313,7 @@ public TransferPair makeTransferPair(ValueVector to) {
   }
 
   private class TransferImpl implements TransferPair {
+
```

Comment:
Needed?

Reply:
I guess it's added by my IDE. I'll revert it.